README.md

```diff
@@ -31,15 +31,15 @@ your answer to this markdown file.
 
 ANSWER:  
 I believe that a good invariant for this would be that at the beginning of every recursive call  
-the previous two terms will *always* be available. The base case is true because before we began the recursive call, we  
+the previous two terms in the Fibonacci sequence will *always* be available. The base case is true because before we began the recursive call, we  
```

Comment:
When you say "available", do you mean "correct"? Please be specific with your wording.